### ISSUE TITLE
Allow searching for journal names

### ISSUE THREAD

**Issue by Samwalton9** [Samwalton9/WikiCiteVis#12]
We could allow searching for journal names (as strings), mapping to their DOI and returning results for it, in the same way Cite-o-meter does. We could even borrow their doi_pref file: https://github.com/wpoa/Wikipedia-Cite-o-Meter/blob/master/doi_pref.tsv

**Comment by Daniel-Mietchen** [comment]
I don't think that old file will help you much. We got it from CrossRef, where the data should nowadays be more dynamically accessible (pinging @afandian).

You might also consider using Wikidata (or its autosuggest/autocomplete) for searching.
